data/assets/actions/trails/trail_appearance.asset

```diff
@@ -90,40 +86,28 @@ else
   local prop = list[1];
   local fadeValue = openspace.getPropertyValue(prop)
   if fadeValue > 1 then
-    ]] .. getHightlightCommand(identifierString, 1, nameString) .. "\n" .. [[
+    ]] .. highlightCommand(identifierString, 1, nameString) .. "\n" .. [[
   else
-    ]] .. getHightlightCommand(identifierString, value, nameString) .. "\n" .. [[
+    ]] .. highlightCommand(identifierString, value, nameString) .. "\n" .. [[
   end
 end]],
     Documentation = "Animates the trail fade of " .. nameString .. "'s trail",
     GuiPath = "/Trails/Appearance",
     IsLocal = true
   }
-  return action
 end
 
-local earthTrailIdentifer = "Scene.EarthTrail"
-
-local highlight_earth_trail = getHighlightAction(earthTrailIdentifer, 10, "Earth")
-asset.export("highlight_earth_trail", highlight_earth_trail.Identifier)
-local dehighlight_earth_trail = getHighlightAction(earthTrailIdentifer, 1, "Earth")
-asset.export("dehighlight_earth_trail", dehighlight_earth_trail.Identifier)
-local toggle_earth_trail_highlight = getToggleHighlightAction(earthTrailIdentifer, 10, "Earth")
-asset.export("toggle_earth_trail_highlight", toggle_earth_trail_highlight.Identifier)
+local highlight_earth_trail = highlightAction("Scene.EarthTrail", 10, "Earth")
```

Comment:
Should variables like these be camel case? Or did we decide that we don't care for variable names?

Reply:
we seem to have changed all the "regular ones", but not the ones that are returned from functions?

Reply:
No, they should all be camelCase or CamelCase